lib/Backends/CMakeLists.txt

```diff
@@ -1,5 +1,11 @@
 add_library(Backends Backends.cpp)
 
+add_library(BackendUtils BackendUtils.cpp)
+
+target_link_libraries(BackendUtils
+                      PRIVATE
+                      IR)
```

Comment:
can you make sure there is no circular dependency introduced here?

you can run `ninja dependency_graph` and check .dot file.

Reply:
Checked the Dot file, No circular dependency introduced. IR (introduced by BackendUtils) does not depend on anything upstream.